### ISSUE TITLE
[RFC] Features you want to see

### ISSUE THREAD

**Issue by ma7dev** [approx-ml/approx#11]
In this issue, you are welcome to discuss features that you want to see included in the library.

**Comment by ma7dev** [comment]
The currently supported operations for Dynamic Quantization based on [1](https://pytorch.org/docs/stable/quantization.html#operator-support) are:
- [x] `nn.Linear`
- [ ] `nn.LSTM`
- [ ] `nn.GRU`
- [x] `nn.RNNCell`
- [x] `nn.GRUCell`
- [x] `nn.LSTMCell`
- [x] `nn.EmbeddingBad`

We might need to look into Static Quantization as they provide coverage for `nn.Conv1d/2d/3d`, `nn.Embedding`, and `Activations` [1](https://pytorch.org/docs/stable/quantization.html#operator-support).

Another idea is that `FX Graph mode` will be the go-to for future development but it is under construction, so we might be able to contribute to PyTorch and utilize it in our library [2](https://pytorch.org/tutorials/prototype/fx_graph_mode_quant_guide.html).

- [Introduction to Quantization on PyTorch](https://pytorch.org/blog/introduction-to-quantization-on-pytorch/)
- [(PROTOTYPE) FX GRAPH MODE QUANTIZATION USER GUIDE](https://pytorch.org/tutorials/prototype/fx_graph_mode_quant_guide.html)
- [(PROTOTYPE) FX GRAPH MODE POST TRAINING STATIC QUANTIZATION](https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_static.html)
- [(PROTOTYPE) FX GRAPH MODE POST TRAINING DYNAMIC QUANTIZATION](https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_dynamic.html)
- [DYNAMIC QUANTIZATION](https://pytorch.org/tutorials/recipes/recipes/dynamic_quantization.html) - testing model
    - Model size (when saving the model in the disk, what its size)
    - Latency (speed to run forward pass)
    - Accuracy
- [Quantization Triage](https://github.com/pytorch/pytorch/projects/17) - hunting ground
